src/main/java/com/twilio/ivrrecording/servlet/WebAppServlet.java

```
@@ -1,22 +1,28 @@
 package com.twilio.ivrrecording.servlet;
 
-import com.twilio.sdk.verbs.TwiMLResponse;
+import java.io.IOException;
 
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletResponse;
-import java.io.IOException;
+
+import com.twilio.twiml.TwiMLException;
+import com.twilio.twiml.VoiceResponse;
 
 public class WebAppServlet extends HttpServlet {
 
-    protected WebAppServlet() {
-    }
+  protected WebAppServlet() {}
 
-    protected void respondTwiML(HttpServletResponse response, TwiMLResponse twiMLResponse) throws IOException {
-        response.setContentType("text/xml");
-        response.getWriter().write(twiMLResponse.toXML());
+  protected void respondTwiML(HttpServletResponse response, VoiceResponse twiMLResponse)
+      throws IOException {
+    response.setContentType("text/xml");
+    try {
+      response.getWriter().write(twiMLResponse.toXml());
+    } catch (TwiMLException e) {
+      System.out.println("Unable to create TwiML");
```

I would prefer to re-throw the exception. ;)